documentation/ci-fix.md

```diff
@@ -130,20 +130,16 @@ Then refer to [this TSG](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/in
-
-| Error Code                   | Severity | Solution                                                                                                                                                                                                                                                                                     |
-|------------------------------|----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| MissingTypeSpecFile          | Error    | Adding the related TypeSpec project into {RP-Name} folder, like [Qumulo.Management](https://github.com/Azure/azure-rest-api-specs/tree/main/specification/liftrqumulo/Qumulo.Management)                                                                                                     |
-| MissingExamplesDirectory     | Error    | The example files should be kept in the 'examples' folder under the TypeSpec project,the typespec-autorest emitter will copy them into the output folder and create corresponding 'x-ms-examples' in the swagger automatically when geneates the swagger, you should also check in it in PR. |
-| InConsistentSwagger          | Error    | The generated swagger is inconsistent with the swagger in PR, so you need to re-generate swagger from TypeSpec project and check in it.                                                                                                                                                      |
-| SwaggerNotExistInPR          | Error    | It occurs when there is a TypeSpec file in the PR but the generated swagger is not present in the PR, so you need to add the swagger to the PR.                                                                                                                                              |
-| GeneratedSwaggerNotFound     | Error    | It occurs when there is a TypeSpec file in the PR but there is no swagger produced by the typespec-autorest emitter, so you need to check the tspconfig.yaml to see if it has the wrong configuration, like 'output-dir' or 'azure-resource-provider-folder'.                                |
-| MissingTypeSpecProjectConfig | Warning  | The configuration of '@azure-tools/typespec-autorest' including 'output-file','azure-resource-provider-folder' are used to customize the generated swagger file name and folder structure, it's recommended to use them in the 'tspconfig.yaml', here is a [sample tspconfig.yaml].          |
-
-See [typespec-autorest](https://azure.github.io/typespec-azure/docs/emitters/typespec-autorest) for more information about how the swagger is generated from the TypeSpec project.
+### Run tsv locally
+```
+npm install
+npx --no tsv <path to project folder>
+# example:
+npx --no tsv specification/contosowidgetmanager/Contoso.WidgetManager
```

Comment:
```suggestion
$ cd <repo>
$ git checkout <your-branch>
$ git merge <target-branch>
$ npm ci
$ npx tsv <path-to-your-spec>
$ git commit; git push (if any changes)

# example:
$ npx tsv specification/contosowidgetmanager/Contoso.WidgetManager
```

Reply:
Why add the generic git info about checkout and merging? Do folks you have been helping not understand git?

Reply:
In my experience, many spec authors seem unfamiliar with git, especially concepts like branching and merging.  A common problem is PRs failing TSV since the branch has been open for months and is not regularly merging from target.  So we definitely need include "merge latest from target before running locally" in our instructions.

If we want spec authors to fix TSV without needing to contact us, I think we need to provide very clear instructions, ideally commands they could copy/paste to their local machine.

Reply:
I'm good with this here. We might actually want to consider adding a section about handling git specific scenarios in another point in time.